.github/ISSUE_TEMPLATE/--weekly-check-in.md

```diff
@@ -1,3 +1,12 @@
+---
```

Comment:
Thanks for noting this...we merged a PR that added this #7949, maybe you could rebase with the master to get this changes...You are doing amazing work  :tada:

Reply:
Thanks a ton!! :tada: And thank you for  helping all of us out :clap: :100: and the numerous code reviews :v: